.github/workflows/build-k0s.yml

```diff
@@ -71,14 +71,14 @@ jobs:
           make bindata
           make --touch codegen
           make build
+          echo "k0s${{ steps.build-prepare.outputs.executable-suffix }} binary size: **$(du -sh k0s${{ steps.build-prepare.outputs.executable-suffix }} | cut -f1)**" >>$GITHUB_STEP_SUMMARY
```

Comment:
can we somehow post it as a comment to the PR?

Reply:
We can, but it require us to run a proxy service in our infra.
https://github.com/marketplace/actions/add-pr-comment#proxy-for-fork-based-prs